automat/_test/test_visualize.py

```diff
@@ -44,8 +62,142 @@ def out(self):
     return mm
 
 
+class ElementMakerTests(TestCase):
+    """
+    Tests that ensure elementMaker generates correct HTML.
```

Comment:
"correct" is a word we specifically try to avoid in tests: see https://jml.io/pages/test-docstrings.html